### PR TITLE
Cadl compiler 0.34

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
error

```
cadl compile ./http/models/collections-basic/main.cadl
Cadl compiler v0.34.0

Diagnostics were reported during compilation:

C:/github/autorest.java/cadl-tests/http/models/collections-basic/main.cadl:7:7 - error invalid-ref: Namespace C:/github/autorest.java/cadl-tests/node_modules/@azure-tools/cadl-azure-core/dist/src/index.js.Azure.Core doesn't have member Operations
> 7 | using Azure.Core.Operations;
    |       ^^^^^^^^^^^^^^^^^^^^^
C:/github/autorest.java/cadl-tests/http/models/collections-basic/main.cadl:50:66 - error unknown-identifier: Unknown identifier ErrorResponse
> 50 | op sendCollectionModel(@body input: InputModel): Response<200> | ErrorResponse;
     |                                                                  ^^^^^^^^^^^^^
C:/github/autorest.java/cadl-tests/http/models/collections-basic/main.cadl:56:26 - error invalid-template-args: Can't pass template arguments to non-templated type
> 56 | op getCollectionModel(): OkResponse<OutputModel> | ErrorResponse;
     |                          ^^^^^^^^^^^^^^^^^^^^^^^
C:/github/autorest.java/cadl-tests/http/models/collections-basic/main.cadl:56:52 - error unknown-identifier: Unknown identifier ErrorResponse
> 56 | op getCollectionModel(): OkResponse<OutputModel> | ErrorResponse;
     |                                                    ^^^^^^^^^^^^^
C:/github/autorest.java/cadl-tests/http/models/collections-basic/main.cadl:62:53 - error invalid-template-args: Can't pass template arguments to non-templated type
> 62 | op setCollectionModel(@body input: RoundTripModel): OkResponse<RoundTripModel> | ErrorResponse;
     |                                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^
C:/github/autorest.java/cadl-tests/http/models/collections-basic/main.cadl:62:82 - error unknown-identifier: Unknown identifier ErrorResponse
> 62 | op setCollectionModel(@body input: RoundTripModel): OkResponse<RoundTripModel> | ErrorResponse;
     |                                                                                  ^^^^^^^^^^^^^

Found 6 errors.
```

### CODE REVIEW

File: cadl-tests/generate.sh [hunk 1]
```diff
@@ -27,10 +27,12 @@ do
...
 
-# run cadl ranch tests sources
-for f in $(find ./node_modules/@azure-tools/cadl-ranch-specs/http -name "*.cadl")
-do
-  if [[ $(realpath $f) == *"models"* || $(realpath $f) == *"hello"* ]]; then
-    generate $f
-  fi
-done
+## run cadl ranch tests sources
+#cp -rf node_modules/@azure-tools/cadl-ranch-specs/http .
```
Comment (weidongxu-microsoft) [inline cadl-tests/generate.sh:31]: If called in `node_modules`, cadl compiler might find that they are not using same version of compiler.

File: cadl-tests/generate.sh [hunk 2]
```diff
@@ -27,10 +27,12 @@ do
...
+#for f in $(find ./http -name "*.cadl")
+#do
+#  if [[ $(realpath $f) == *"models"* ]]; then
```
Comment (weidongxu-microsoft) [inline cadl-tests/generate.sh:34]: the `hello` no longer works in 0.34, as `getAllRoutes` return empty.

**Comment by weidongxu-microsoft** [comment]
@haolingdong-msft @XiaofeiCao Please rest cadl-extension in your local, after this PR.
https://github.com/Azure/autorest.java/blob/main/cadl-tests/setup.sh#L3-L4

File: cadl-tests/generate.sh (hunk 2 above)
Comment (haolingdong-msft) [reply cadl-tests/generate.sh:34]: OK, we can generate [srv-driven-2](https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/resiliency/srv-driven-2) code and test.

Comment (weidongxu-microsoft) [reply cadl-tests/generate.sh:34]: Sure, if some of them works, you can add them back.

**Pull request merged**